frontend/talentsearch/src/js/components/experienceForm/ExperienceForm.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import type { Meta, Story } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import type { Skill } from "@common/api/generated";
+import { fakeSkills, fakeSkillFamilies } from "@common/fakeData";
+import { ExperienceForm } from "./ExperienceForm";
+import type { ExperienceType } from "./types";
+
+const skillData = fakeSkills(15, fakeSkillFamilies(4));
+
+export default {
+  component: ExperienceForm,
+  title: "ExperienceForm",
+  args: {
+    experienceType: "award",
+    skills: skillData,
+  },
+} as Meta;
+
+interface ExperienceFormStoryArgs {
+  experienceType: ExperienceType;
+  skills: Skill[];
+}
+
+const TemplateExperienceFormForm: Story<ExperienceFormStoryArgs> = ({
+  experienceType,
+  skills,
+}) => (
+  <ExperienceForm
+    experienceType={experienceType}
+    skills={skills}
+    onUpdateExperience={async (data: any) => {
```

Comment:
Should this `any` be `ExperienceDetailsSubmissionData`?

Reply:
Resolved with [annotate experience form update callback args](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2480/commits/bbafac2361357809c10c35c86e0664b2ea146da7)